about.html

```diff
@@ -0,0 +1,108 @@
+        <ul class="first-desktop-nav">
+          <li><i class="fa fa-facebook" aria-hidden="true"></i></li>
+          <li><i class="fa fa-twitter" aria-hidden="true"></i></li>
+          <li>English</li>
+          <li>My Page</li>
+          <li>Logout</li>
+        </ul>
+        <div class="nav-logo">
+          <a href="./index.html"><img src="./assets/images/logo_1.png" alt="logo"></a>
+          <ul class="second-desktop-nav">
+            <li><a href="#">About</a></li>
+            <li><a href="#">Program</a></li>
+            <li><a href="#">Join</a></li>
+            <li><a href="#">Sponsor</a></li>
+            <li><a href="#">News</a></li>
+          </ul>
+        </div>
+      </nav>
+      <div class="header-title">
+        <h2>"Hello, Sharing world!"</h2>
+        <h1>Digital Nomads<br>Global Summit<br>2022</h1>
+        <hr class="red-line-desktop">
+      </div>
+      <p class="header-paragraph">Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula eget
+        dolor. Aenean massa. Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Donec
+        quam felis, ultricies nec, pellentesque eu, pretium quis, sem. Nulla consequat massa
+      </p>
+      <P class="main-email">Please contact us per Email for any further questions<br>about Digital Nomads Global Summit
+        2022!<br><a href="#">dnsummit2022@dn.org</a></P>
+
+    </header>
+    <main>
+      <section class="logo">
+        <h2 class="logo-title">Digital Nomads<br>Gloval Summit 2022 Logo</h2>
+        <hr class="red-line">
+        <p class="logo-paragraph">The logo of Digital Nomads Global Summitt 2022 was decided through the logo
+          competition
+          from 8.June to .July.</p>
+        <img src="./assets/images/logo_1.png" class="image-logo" alt="logo">
+        <hr class="logo-grey-line">
+      </section>
+      <section class="past">
+        <h2 class="past-title">See the past Global Summits</h2>
+        <hr class="red-line">
+        <p class="past-paragraph">Take a look at the two last Global Summits which took place in Buenos Aires and
+          Varsovia
+        </p>
+        <div class="past-images">
+          <img src="./assets/images/past_2011.png" class="past-image" alt="summit 2011">
+          <img src="./assets/images/past_2013.png" class="past-image" alt="summit 2013">
+        </div>
+
+      </section>
+    </main>
+
+    </section>
+    <section class="sponsors">
+      <h3 class="sponsors-title">Partner</h3>
+      <hr class="red-line">
+      <ul class="all-sponsors">
+        <li><img src="./assets/images/mozilla.png" alt="Mozilla"></li>
+        <li><img src="./assets/images/google.png" alt="Google"></li>
+        <li><img src="./assets/images/naver.png" alt="Naver"></li>
+        <li><img src="./assets/images/daumkakao.png" alt="Daumkakao"></li>
+        <li><img src="./assets/images/airbnb.png" alt="Airbnb"></li>
+      </ul>
+    </section>
+  </div>
```

Comment:
- [ ] It appears that after the sponsor's section on the mobile view you might have forgotten to add the footer section. Kindly include it in your project to make the design look complete.

**This is the footer design:**

![image](https://user-images.githubusercontent.com/86318284/196673329-d1c40f7c-cf71-4c8c-b44f-ba4eec77c68d.png)